src/lib.rs

```diff
@@ -1063,7 +1063,7 @@ impl CommandEncoder {
 
 impl<'a> RenderPass<'a> {
     /// Sets the active bind group for a given bind group index.
-    pub fn set_bind_group(&mut self, index: u32, bind_group: &BindGroup, offsets: &[u32]) {
+    pub fn set_bind_group(&mut self, index: u32, bind_group: &BindGroup, offsets: &[u64]) {
```

Comment:
Can we use the `BufferAddress` type for the offsets? It's supposed to be imported from `wgpu-native`